.gitignore

```diff
@@ -10,6 +10,9 @@
 # ignore local arch files not meant to be shared
 /arch/local*
 
+# ignore manual generated locally
+/tools/manual/*
```

Comment:
Unfortunately, this is too general because it exclucdes the entire [tools/manual](https://github.com/cp2k/cp2k/tree/master/tools/manual) directory. AFAIK, we don't have a default output dir. So, there is probably no sensible rule we could add here.

Reply:
Thanks for the comment.

Reply:
I have removed the local manual. Please check.